docs/src/_includes/components/nav-version-switcher.html

```diff
@@ -1,7 +1,7 @@
 <div class="version-switcher">
     <a href="/versions/" class="switcher-fallback">Versions</a>
 
-    <div hidden role="region" class="switcher switcher--version" aria-labelledby="nav-version-switcher-label" id="nav-version-switcher">
+    <div  role="region" class="switcher switcher--version" aria-labelledby="nav-version-switcher-label" id="nav-version-switcher">
```

Comment:
This fixes a layout shift that happens only on mobile?

Reply:
yes

Reply:
We'll have the same problem here, users without JavaScript will see a select element that doesn't work.

This is inside [`nav-panel`](https://github.com/eslint/eslint/blob/10249ad1a961463b6b347be71c074951ab8e2652/docs/src/_includes/components/navigation.html#L12), which itself causes a layout shift on mobile. Would it be better to set `data-open="false"` initially on `nav-panel`?

Reply:
> This is inside [`nav-panel`] Would it be better to set `data-open="false"` initially on `nav-panel`?

This actually won't work well when JavaSript is disabled because there will be no way to show the panel.

Reply:
@mdjermanovic I tried the same `<noscript><style>...</style></noscript>`